docs/shared/register-federated-cli.mdx

```diff
@@ -1,53 +1,42 @@
-In a federated architecture, each of your graph's [implementing services](https://www.apollographql.com/docs/apollo-server/federation/implementing-services/) uses the Apollo CLI to register its schema as a separate service **of the same graph**:
+In a federated architecture, each of your graph's [subgraphs](https://www.apollographql.com/docs/federation/implementing-services/) uses the Rover CLI to register its schema with Apollo:
```

Comment:
Any chance we could change the URL to reflect "subgraphs" too?

Reply:
This will be in a big unwieldy followup PR that just replaces "implementing service" with "subgraph" all over the place